MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include pringles/backends/statics *
```

Comment:
whats this??

Reply:
See https://github.com/colonelpringles/pringles/pull/55#discussion_r322025149